tripy/tests/integration/test_sequential.py

```diff
@@ -0,0 +1,90 @@
+#
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import torch
+import tripy as tp
+
+class TestSequential:
+    def test_forward_pass_accuracy(self):
+        torch_model = torch.nn.Sequential(
+            torch.nn.Linear(1, 3, dtype=torch.float32), torch.nn.Linear(3, 2, dtype=torch.float32)
+        )
+        tp_model = tp.Sequential(tp.Linear(1, 3, dtype=tp.float32), tp.Linear(3, 2, dtype=tp.float32))
+
+        tp_model[0].weight = tp.Parameter(torch_model[0].weight.detach())
+        tp_model[0].bias = tp.Parameter(torch_model[0].bias.detach())
+        tp_model[1].weight = tp.Parameter(torch_model[1].weight.detach())
+        tp_model[1].bias = tp.Parameter(torch_model[1].bias.detach())
+
+        input_tensor = torch.tensor([[1.0]], dtype=torch.float32).to('cuda')
```

Comment:
Let's just set the device in the constructor so we don't need to do the copy

Reply:
done